src/tests/system/tests/test_passkey.py

```diff
@@ -26,6 +26,17 @@ def passkey_requires_root(client: Client) -> tuple[bool, str] | bool:
     return True
 
 
+@mh_fixture()
```

Comment:
You can omit parentheses.

Reply:
mh_fixture is not imported. You need to add `from pytest_mh import mh_fixture`

Reply:
updated.